### DISCUSSION TITLE
display annotations while using OS

### DISCUSSION THREAD

**Discussion by D2OR** [pubpub-zz/ppInk#190] [Q&A]
Wonderful software, I think. Useful to create annotations that would not be clickable (be ignored) by the OS.
Better than the EpicPen.
However, I cannot figure out how to return to the "MOUSE MODE" for longer than one click.
When I create my annotation, I need to go back to the normal windows operations while keeping the annotations visible, yet ignored by the OS, if I click on them by accident or drag them.
I figured this is what the mouse mode is for. Unfortunately, after I click the mouse mode button, I am only able to click just once to operate the OS, before the ppInk returns to the drawing mode again.

How can I configure it to stay in the "MOUSE MODE" until I am done with sometimes many OS actions and able or need to return to the drawing mode to adjust or turn off the annotations? I don't need anything but the annotations themselves to be visible (and ignored by the OS) while I work. The annotations would be highlighting the important areas that need to be monitored / paid attention to, but I must continue to work in windows. If I have to go back into ppInk after ever single mouse click or keep some key pressed that would make me inefficient while operating the OS.

Please, help me figure this out or point me to a help file that contains this explanation. I can't find this.

**Comment by pubpub-zz** [comment]
Hi @D2OR,
Happy you like it(do not hesitate to star it).
the pointer mode/mouse mode( engaged with ![image](https://user-images.githubusercontent.com/4083478/174470831-bcf24f0e-912f-410b-ab3d-e37ceec8ec8f.png) ) stays active until the global hotkey is depressed, or clicking on the undocking button shortcut
I remember some users reported having some interferences due to other program. You should review your active programs
